independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/config/json/JsonRegistryConfig.java

```diff
@@ -46,13 +46,14 @@ public void setId(String id) {
         this.id = Objects.requireNonNull(id);
     }
 
+    @JsonInclude(value = JsonInclude.Include.CUSTOM, valueFilter = JsonBooleanTrueFilter.class)
```

Comment:
I still don't get why this is needed

Reply:
This is applying the the filter. Is there something redundant?

Reply:
I don't get why do you need `@JsonInclude` and a filter here? The filter doesn't seem to do anything special?

Reply:
Oh, it does. W/o it it will be always serializing `enabled: true`. With the filter it switches the default for the boolean that should be skipped during serialization.